add-on/chrome/bootstrap.js

```diff
@@ -671,10 +683,17 @@ var WindowListener = {
           null,
           box.PRIORITY_WARNING_LOW,
           [{
-            label: initStrings.label,
+            label: this._roomSessionConnected ? initStrings.label : pendingBtnString,
             accessKey: initStrings.accessKey,
             isDefault: false,
             callback: (event, buttonInfo, buttonNode) => {
+              // Prevent any action before connecting to a room.
+              if (!this._roomSessionConnected) {
+                // You must throw an error to prevent the notification bar
+                // from closing when the button is clicked.
+                throw new Error('Prevent notification bar close');
+              }
```

Comment:
The error is thrown so that the button will not close the window 
https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/Method/appendNotification
The API specifies that.
Changing the UI (from a disabled button to an active one) doesn't help much (difference is very small) it is barely noticeable. Showing the button after the connection is established might look odd and editing the buttons in the notification bar is potentially difficult. You would need a reference to the notification bar created (please correct me if I'm wrong).